launch/src/logger.rs

```diff
@@ -23,7 +25,7 @@ pub fn init_logger() {
         .with_thread_ids(false) // set to true to display id of the thread emitting the log
         .with_source_location(true) // set to true to include source file and line number in log
         .with_target(false) // set to true to include module name in logs
-        .with_ansi(true) // set to false to remove color in output
+        .with_ansi(false) // set to false to remove color in output
```

Comment:
Most of the time, we do use logs on our computers and color is useful. Is there a reason for this change? Or maybe we can do something like this?
```suggestion
        .with_ansi(std::env::var("LOKI_ANSI_COLOR").and_then(bool::from_str).unwrap_or(true)) // set to false to remove color in output
```

Reply:
Done :+1: